actionview/lib/action_view/template/handlers/erb.rb

```diff
@@ -58,6 +58,7 @@ def add_expr_escaped(src, code)
         def add_stmt(src, code)
           flush_newline_if_pending(src)
           super
+          flush_newline_if_comment(src, code)
```

Comment:
> would anything break if `add_stmt` simply always did `src << "\n"`?

Yes: that would break line number alignment.

If we, for whatever reason, only care about the effects of such a comment in the end-of-file position, however, I would think having `add_postamble` always insert a newline should be fine: by definition, it comes after the end of the file we're trying to match lines with.